src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/models/ReportRun.kt

```diff
@@ -1,3 +1,19 @@
 package org.vaccineimpact.orderlyweb.models
 
+import java.time.Instant
+
 data class ReportRun(val name: String, val key: String, val path: String)
+
+data class ReportRunLog
+constructor(
+    val email: String,
+    val date: Instant,
+    val report: String,
+    val instances: String?,
```

Comment:
I ca see that you're doing the JSON parsing of instances and params in the front end, but would it not be nicer to rehydrate them in the backend, so that we're not sending back some weird JSON response which includes JSON strings? That way, the instances and params structure could be expressed in this backend class, and the data from the response could be used as a front end ReportLog object immediately without the front end having to do any further processing.